modules/installation-vsphere-machines.adoc

```diff
@@ -62,30 +62,29 @@ endif::openshift-origin[]
 .. Click the *VMs and Templates* view.
 .. Right-click the name of your datacenter.
 .. Click *New Folder* -> *New VM and Template Folder*.
-.. In the window that is displayed, enter the folder name. If you did not specify an existing folder in the `install-config.yaml` file, then create a folder with the same name as the infrastructure ID.
+.. In the window that is displayed, enter the folder name. If you did not specify an existing folder in the `install-config.yaml` file, create a folder with the same name as the infrastructure ID.
 
-. In the vSphere Client, create a template for the OVA image.
+. In the vSphere Client, create a template for the OVA image and then clone the template as needed.
 +
 [NOTE]
 ====
-In the following steps, you use the same template for all of your cluster machines and provide the location for the Ignition config file for that machine type when you provision the VMs.
+In the following steps, you create a template and then clone the template for all of your cluster machines. You then provide the location for the Ignition config file for that cloned machine type when you provision the VMs.
 ====
 .. From the *Hosts and Clusters* tab, right-click your cluster name and select *Deploy OVF Template*.
 .. On the *Select an OVF* tab, specify the name of the {op-system} OVA file that you downloaded.
-.. On the *Select a name and folder* tab, set a *Virtual machine name*, such as {op-system}. Click the name of your vSphere cluster and select the folder you created in the previous step.
+.. On the *Select a name and folder* tab, set a *Virtual machine name* for your template, such as `Template-{op-system}`. Click the name of your vSphere cluster and select the folder you created in the previous step.
 .. On the *Select a compute resource* tab, click the name of your vSphere cluster.
 .. On the *Select storage* tab, configure the storage options for your VM.
 *** Select *Thin Provision* or *Thick Provision*, based on your storage preferences.
 *** Select the datastore that you specified in your `install-config.yaml` file.
 .. On the *Select network* tab, specify the network that you configured for the cluster, if available.
-.. If you plan to use the same template for all cluster machine types, do not specify values on the *Customize template* tab.
+.. When creating the OVF template, do not specify values on the *Customize template* tab or configure the template any further.
 +
 [IMPORTANT]
 ====
-If you plan to add more compute machines to your cluster after you finish
-installation, do not delete this template.
+Do not start the original VM template. The VM template must remain off and must be cloned for new {op-system} machines. Starting the VM template configures the VM template as a VM on the platform, which prevents it from being used as a template that machine sets can apply configurations to.
```

Comment:
Does this step, Step 4, am I starting the template here? I'm a bit scared as I don't know how or when to start the templates, or how to make sure it's off.

Reply:
No, you are not starting it in step 4; it is off at creation. You are only creating the template in this step and the note is saying not to start it. (Not exactly how one "starts" the VM in vCenter.)